frontend/src/i18n/locales/en.yaml

```
@@ -302,13 +302,24 @@ manage:
       user: User
       yourself: You
       subset-warning: 'This selection is already included in the following group(s): {{groups}}.'
+      inherited: Inherited permissions
+      inherited-tooltip: The following entries can only be changed on the page they originate from.
       actions:
         title: Actions
         read: Read
         read-description: Can watch the video and view metadata.
         write: Read and write
-        large-group-warning: You are giving write-access to a large group. Is this intentional?
         write-description: Can also edit the video, metadata and other details.
+        moderate: Moderate
+        moderate-description: Can add subpages and edit settings and content.
```

```suggestion
        moderate-description: Can edit content, add subpages and change the name & subpage order.
```

Is a bit more explicit?